miden-crypto/src/hash/rescue/rpo/mod.rs

```diff
@@ -1,15 +1,12 @@
 use core::ops::Range;
 
 use super::{
-    ARK1, ARK2, BINARY_CHUNK_SIZE, CAPACITY_RANGE, DIGEST_BYTES, DIGEST_RANGE, DIGEST_SIZE, Digest,
-    ElementHasher, Felt, FieldElement, Hasher, INPUT1_RANGE, INPUT2_RANGE, MDS, NUM_ROUNDS,
-    RATE_RANGE, RATE_WIDTH, STATE_WIDTH, StarkField, ZERO, add_constants,
-    add_constants_and_apply_inv_sbox, add_constants_and_apply_sbox, apply_inv_sbox, apply_mds,
-    apply_sbox,
+    ARK1, ARK2, BINARY_CHUNK_SIZE, CAPACITY_RANGE, DIGEST_RANGE, ElementHasher, Felt, FieldElement,
+    Hasher, INPUT1_RANGE, INPUT2_RANGE, MDS, NUM_ROUNDS, RATE_RANGE, RATE_WIDTH, STATE_WIDTH,
+    StarkField, ZERO, add_constants, add_constants_and_apply_inv_sbox,
+    add_constants_and_apply_sbox, apply_inv_sbox, apply_mds, apply_sbox,
 };
-
-mod digest;
-pub use digest::{RpoDigest, RpoDigestError};
+pub use crate::{Word, WordError};
```

Comment:
Related to the previous comment: I'm not sure it makes sense to re-export these from here since they are already exported from the root of the library.

Reply:
Oops yes this is a left over of search-replace. Good catch